Frontend/src/Components/PPC_MPPT/MPPT_CELL.js

```diff
@@ -16,10 +16,9 @@ export default function MPPT_CELL(props){
             bg={props.isCurrMode ? greenBgCol : redBgCol}
             borderColor={borderCol}
             borderWidth='2px'
-            lineHeight='1.6em'
+            lineHeight={props.lineHeight}
         >
-            <Text as='b' fontSize='1.3vw' color={props.isCurrMode ? greenTxtCol : redTxtCol}>{props.label}</Text>
+            <Text align={"center"} as='b' fontSize={props.fontSize} color={props.isCurrMode ? greenTxtCol : redTxtCol}>{props.label}</Text>
```

Comment:
```suggestion
            <Text align='center' as='b' fontSize={props.fontSize} color={props.isCurrMode ? greenTxtCol : redTxtCol}>{props.label}</Text>
```
nit: {} aren't needed around strings. Also, for consistency, we're using single quotes around prop values in this file.

Reply:
fixed